pages/index.md

```diff
@@ -3,32 +3,4 @@ title: 18F Front-End Guide
 permalink: /
 ---
 Welcome to the 18F Front-End Guide! This is where we keep all of our
-guidelines for front-end design and development.
-
-## What is front end?
-
-The Front-End Guild did a series of exercises to determine the
-fundamental differences between the front-end design and front-end
-developer roles at 18F. Using [some of our own research
-methods](https://methods.18f.gov), the Front-End Guild came up with
-the following recommendations on knowing the difference between the
-two disciplines:
-
-**Front-end designers** design, write, and implement the
-presentational code base for websites and applications. They should
-have a clear understanding of design fundamentals and systems, such
-as interface style guides, responsive design, grid systems, front-end
-frameworks, and accessibility best practices. Front-end designers
-should feel comfortable creating and implementing design systems
-using semantic HTML5, CSS/Sass and be able to assist in debugging
-this aspect of the code base.
-
-**Front-end developers** architect, write, and implement the
-functional code base for websites and applications. They should have
-a clear understanding of client-side render and response, such as
-HTTP methods, API consumption, the browser loading/rendering
-pipeline, and accessibility best practices. Front-end developers
-should feel comfortable developing and implementing client-side
-interactions and frameworks using semantic HTML5 and JavaScript, and
-should be able to help with debugging, testing, and performance
-optimization of the code base.
```

Comment:
I'm torn about deleting this. On one hand, we haven't had front end designers at 18F for nearly two years. On the other hand, it feels valuable to define what we expect from front end developers (and what we don't expect). Perhaps that definition process is one that we should go through again new, though?

Reply:
I think we should keep the bit about front-end developers.